### PR TITLE
Nico initial components

### PR DESCRIPTION
**Pull request by kleyson**
Solving conflicts on @nisnardi code.

Following PR's should contain:
1. Content for Forum, Events, Resource pages;
2. eslint rules and build process;

After that, we are ok to replace the old site with the react version.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,28 +1,32 @@
...
+import Events from './screens/Events';
+import Resources from './screens/Resources';
+import Header from './components/Header'
+import Footer from './components/Footer'
 
+const App = () => {
+    return (
+        <div id="body_container">
+            <Router>
+                <>
```
Comment (samritbk) [inline src/App.jsx]: Is this an empty component to only used to wrap the details? If so it looks good.

Comment (samritbk) [reply src/App.jsx]: Short syntax for React.Fragment, nice to learn new staff.

Comment (kleyson) [reply src/App.jsx]: Yeap, that is a short syntax for `<Fragment />`, and yes it's just wrapping because the Router requires just one child.

Comment (dbernar1) [reply src/App.jsx]: Yay! Didn't know this existed. Thanks.

Comment (dbernar1) [reply src/App.jsx]: I mean, didn't know about Fragment at all.

**Pull request merged**